drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -760,6 +760,13 @@ func getPXNodes(destKubeConfig string) ([]corev1.Node, error) {
 	return pxNodes, nil
 }
 
+func (k *k8sSchedOps) MarkPXNodeForRemoval(n node.Node) error {
```

Comment:
who's calling this?

Remove `if err := k8s.Instance().AddLabelOnNode(n.Name, "px/enabled", "remove"); err != nil {` from the portworx driver and call this instead.